src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -121,9 +121,10 @@ export default defineComponent<Props>({
             type: String as PropType<WidgetSize>,
             default: WIDGET_SIZE.md,
         },
+        // FIXME:: width should be -= 16 because of margin.
         width: {
-            type: String,
-            default: '30rem', // default width of md size
+            type: Number,
+            default: 320, // default width of sm size
```

Comment:
How about this?! I guess it can replace the comment.

```suggestion
            default: WIDGET_FRAME_WIDTH_RANGE_LIST.SM[0]
```

Reply:
GREEEEAAAT!!!